uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -54,9 +54,44 @@ define([
           }.bind(this));
       }
 
+      /**
+       * Save contacts which do not have an ID
+       *
+       * @return {Promise}
+       */
+      function manageComments() {
```

Comment:
how about createComments or updateComments or saveCommentWithID, manage is not so clear?